src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
+  const SplitArray = date.split(oldSeparator);
```

Comment:
maybe newDate or oldDate.
And dont start var's name with upper case

Reply:
yes, agree with Ihor, don't start from the upper case and use a better name, use the word in plural form for arrays